pom.xml

```diff
@@ -251,11 +251,6 @@
         <artifactId>slf4j-log4j12</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
-      <dependency>
```

Comment:
this is inside "<dependencyManagement>" section, it it only a declaration of the version, it is not using the jar
see
https://stackoverflow.com/questions/2619598/differences-between-dependencymanagement-and-dependencies-in-maven

Reply:
Yes, I know.  But it's really occurs in my local and do not reproduce after retry.
Well, it is also ambiguous to declare the two.

Reply:
Should I close this pr ?